tools/run_tests/run_tests.py

```diff
@@ -371,6 +371,7 @@ def test_specs(self):
       tests_json = json.load(tests_json_file)
     environment = dict(_FORCE_ENVIRON_FOR_WRAPPERS)
     environment['PYVER'] = '2.7'
+    environment['PYTHONPATH'] = os.path.abspath('src/python/gens')
```

Comment:
This strikes me as a little brittle for the way that it sets the `PYTHONPATH` rather than simply appends to it. Should it look for an existing `PYTHONPATH` in the environment? Or am I being overly careful and fussy? Am I missing something that guarantees that there is no `PYTHONPATH` in the environment before getting to this line?